expensetrcker.py

```
@@ -0,0 +1,83 @@
+import os
+
+DB_FILE = "data.json"
+
+
+
+DB_FILE = "data.json"
+
+def add_expense(amount, category):
+    if os.path.exists(DB_FILE):
+        with open(DB_FILE) as f:
+            expenses=json.load(f)
```

Ruff (0.12.2)

Undefined name `json`

(F821)